packages/extension-task-item/src/task-item.ts

```diff
@@ -68,7 +68,7 @@ export const TaskItem = Node.create<TaskItemOptions>({
         parseHTML: element => {
           const dataChecked = element.getAttribute('data-checked')
 
-          return dataChecked == null || dataChecked === 'true'
```

Comment:
Should we still keep the null check here?

```ts
return dataChecked === null || dataChecked === '' || dataChecked === 'true'
```

Reply:
No I believe we shouldn't.

Case1: `<li>`, `dataChecked` will be `null`,
Case2: `<li data-checked>`, `dataChecked` will be `''`,
Case3: `<li data-checked="true">`, `dataChecked` will be `'true'`,
Case3: `<li data-checked="false">`, `dataChecked` will be `'false'`,

Case1 and case4, `checked` attribute should be recognize as `false`,
Case 2 and case3, `checked` attribute should be recognize as `true`,